mk_verificator/tests/networking/mtu/test_mtu.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "group",
+    utils.get_groups(utils.get_configuration(__file__))
+)
+def test_mtu(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
+    skipped_ifaces = config["skipped_ifaces"]
+    expected_mtu = config["expected_mtu"]
+
+    total = {}
+    failed_ifaces = {}
+
+    network_info = local_salt_client.cmd(group, 'cmd.run',
+                                         ['sudo ls /sys/class/net/'])
+
+    for node, ifaces_info in network_info.iteritems():
+        if 'kvm' in node:
+            kvm_info = local_salt_client.cmd(node, 'cmd.run',
+                                             ["virsh list | grep jse2 | "
+                                              "awk '{print $2}' | "
+                                              "xargs -n1 virsh domiflist | "
+                                              "grep -v br-pxe | grep br- | "
+                                              "awk '{print $1}'"])
+            ifaces_info = kvm_info.get(node)
+        node_name = node.split('-')[0]
```

Comment:
@dkulykovmirantis thank you... hm, that's not node_name actually, that's group_name, as you wish, you may separate this logic in new pool request